fdbmonitor/fdbmonitor_tests.cpp

```diff
@@ -128,15 +128,25 @@ void testPathOps() {
 }
 
 void testEnvVarUtils() {
-	// Ensure validation passes for good inputs
+	// Ensure key-value extraction works
+	std::pair<std::string, std::string> keyValuePair1 = std::make_pair("FOO", "BAR");
```

Comment:
maybe add `const` qualifier, also a shortcut could be `keyValuePair1{"FOO", "BAR"}`

Reply:
Done.